project/Dependencies.scala

```diff
@@ -5,7 +5,7 @@ object Dependencies {
   object V {
     val cats       = "2.4.2"
     val catsEffect = "2.4.0"
-    val circe      = "0.13.0"
+    val circe      = "0.14.0-M4"
```

Comment:
Is this bump necessary for Scala 3?

Reply:
yeah

Reply:
We could use different versions for scala 2 and 3, but I didn't think it mattered much given circe is only used in the examples module.

Reply:
According to the [release notes](https://github.com/circe/circe/releases/tag/v0.14.0-M4), 0.14.0-M4 is backward binary compatible with 0.13

Reply:
ermm yeah I wouldn't feel very confident with this update 

> This milestone release is backward-binary-compatible with 0.13.x on Scala 2, but the eventual 0.14.0 release is not guaranteed to be compatible with any of these milestones.

but since it is only used in examples we should be fine 👍🏾